harness-util/package-util.sh

```diff
@@ -11,18 +11,18 @@ function pkg_src_dir() {
     echo "./${BUILD_DIR_NAME}/${APP_NAME}_${1}_${2}"
 }
 
-# See `pkg_src_dir()`
-# ${3} - the application name
+# ${1}, ${2} - see `pkg_src_dir()`
+# ${3} - the application extension
```

Comment:
It's not "the application extension" is not really a correct term. One may say "the extension of the application launcher".

I am not sure whether such a level of granularity, when you pass not the name of the launcher file, but only its extension is a good thing. Maybe, it is easier for a reader to see that `${3}` is the application launcher file name, which you can then pass as `create_pkg "linux" "x86-64" "${APP_NAME}.sh" "manylinux2010_x86_64"`. I am not insisting on a change, and is curious on what you think. But regardless of what you think about this, please fix the description of the third parameter.

Reply:
I agree with your comments and did it.